src/components/atoms/Button/Button.tsx

```diff
@@ -121,7 +121,7 @@ const SButton = styled.button<IButtonProps>`
     opacity: 0.8;
     color: ${({ styling }) => styling && hoverFontColors[styling]};
     background-color: ${({ styling }) => styling && activeBackgroundColors[styling]};
-    box-shadow: ${({ styling }) => (styling === 'contrastSecondary' ? activeBoxShadows[styling] : 'none')};
+    box-shadow: ${({ styling }) => (styling === 'contrastSecondary' ? activeBoxShadows[styling] : 'none')} !important;
```

Comment:
The `:focus` styles for `box-shadow` below were more specific and overriding these ones ( `:hover` ) , this is ugly but I couldn't think of an alternative solution at the time 🤔